test/utils/EmptinessCheckerSpec.scala

```diff
@@ -43,56 +44,75 @@ class EmptinessCheckerSpec extends UnitSpec {
   "EmptinessChecker" when {
     "empty object" must {
       "return root path as empty" in {
-        EmptinessChecker.findEmptyPaths(Foo()) shouldBe CompletelyEmpty
+        @nowarn("cat=lint-byname-implicit")
```

Comment:
Consider adding "-Wconf:cat=lint-byname-implicit:s" to build.sbt to silence the warning globally (as we have done elsewhere e.g. see [here](https://github.com/hmrc/property-business-api/blob/817a4547606f9024b315571cae12f06544d44061/build.sbt#L39)) rather than having a `@nowarn` everywhere this bit of code is used.